merkle_drop/cli.py

```diff
@@ -260,3 +263,27 @@ def status(jsonrpc: str, merkle_drop_address: str):
         f"{remaining_seconds}"
         f" ({pendulum.now().add(seconds=remaining_seconds).diff_for_humans(absolute=True)})"
     )
+
+
+@main.command(
+    short_help="Compare the Merkle root of an airdrop file with a deployed contracts one."
+)
+@jsonrpc_option
+@merkle_drop_address_option
+@airdrop_file_argument
+def check_root(jsonrpc: str, merkle_drop_address: str, airdrop_file_name: str):
+    click.echo("Calculate Merkle root by airdrop file...")
+    airdrop_data = load_airdrop_file(airdrop_file_name)
+    merkle_root_file = compute_merkle_root(to_items(airdrop_data)).hex()
+    click.echo(f"Merkle root by airdrop file: '{merkle_root_file}'")
+
+    click.echo("Read Merkle root from contract...")
+    web3 = connect_to_json_rpc(jsonrpc)
```

Comment:
Please change the order of contract root and csv root. It would be shitty to wait 1:30 to then realize that you had a typo in your jsonrpc.

Reply:
:heavy_check_mark: